tests/commands_test.py

```diff
@@ -379,3 +381,60 @@ async def test_touch(mcache):
 async def test_close(mcache):
     await mcache.close()
     assert mcache._pool.size() == 0
+
+
+@pytest.mark.asyncio
+async def test_pylibmc_get_helper(mcache_pylibmc):
+    mc_client = pylibmc.Client(['{}:{}'.format(mcache_pylibmc._pool._host,
+                                               mcache_pylibmc._pool._port)])
+
+    key_values: List[List[Any]] = [
```

Comment:
Should probably use `pytest.parameterize()`, so each value is run as a separate test.

Reply:
will that make any difference?

Reply:
It makes tests cleaner, by reducing the code in the test and making it easier to see exactly what is being tested. It is also clearer exactly what failed at a glance, the failing test would appear as something like `test_pylibmc_get_helper["0.5", 0.5]`.

See for example: https://github.com/aio-libs/aiohttp/blob/master/tests/test_cookiejar.py#L283-L310

Reply:
that is no more readable given there's no more context, thinking about this i think the real benefit is that you can see multiple errors at once instead of having to go through fix, run, fix cycles

Reply:
however will be much slower since you have to do the test set-up each time.  So it's definitely not a win-in

Reply:
updated